Formula/atuin.rb

```diff
@@ -0,0 +1,18 @@
+class Atuin < Formula
+  desc "Improved shell history for zsh and bash"
+  homepage "https://github.com/ellie/atuin"
+  url "https://github.com/ellie/atuin/archive/refs/tags/v0.7.1.tar.gz"
+  sha256 "0e1cd2104c6fa4cd2d6f30ae9771ed3f22f78a4e0e7b5243e42abb753b792c38"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "autoload -U add-zsh-hook", shell_output("atuin init zsh")
+    assert_match "", shell_output("atuin history list")
```

Comment:
This doesn't actually check anything

Reply:
Oops, missed that. This might work:

```suggestion
    assert shell_output("atuin history list").blank?
```

Reply:
Yup, this looks like what I was missing!